theme/gatsby-node.js

```diff
@@ -46,10 +46,12 @@ exports.createPages = async ({graphql, actions}, themeOptions) => {
   // Turn every MDX file into a page.
   return Promise.all(
     data.allMdx.nodes.map(async node => {
-      const pagePath = path.join(
-        node.parent.relativeDirectory,
-        node.parent.name === 'index' ? '/' : node.parent.name,
-      )
+      const pagePath = path
+        .join(
+          node.parent.relativeDirectory,
+          node.parent.name === 'index' ? '/' : node.parent.name,
+        )
+        .replace(/\\/g, '/') // Needed to fix paths on windows file systems
```

Comment:
It might be nice to be a little more specific about what this is doing in the comment:
```suggestion
        .replace(/\\/g, '/') // Convert Windows backslash paths to slash paths: foo\\bar → foo/bar
```